db/models/comments.py

```diff
@@ -0,0 +1,138 @@
+import uuid
+from db import db
+from datetime import datetime
+from xml.etree.ElementTree import Comment
+from db.models.assessment import Assessment
+from db.models.sub_criteria import SubCriteria
+from sqlalchemy import DateTime
+from sqlalchemy.exc import IntegrityError
+from sqlalchemy_utils.types import UUIDType
+from sqlalchemy.exc import StatementError
+
+class Comment(db.Model):
+    id = db.Column(
+        UUIDType(binary=False),
+        default=uuid.uuid4,
+        primary_key=True,
+    )
+    comment = db.Column(
+        db.Text(),
+    )
+    created_at = db.Column(
+        DateTime(), 
+        default=datetime.utcnow
+    )
+    assessment_id = db.Column(
+        UUIDType(binary=False),
+        db.ForeignKey(Assessment.id),
+    )
+    assessor_user_id = db.Column(
+        db.Text(),
+    )
+    sub_criteria_id = db.Column(
+        UUIDType(binary=False),
+        db.ForeignKey(SubCriteria.id),
+        nullable=False,
+    )
+    section_id = db.Column(
+        db.Integer(),
+        nullable=False
+    )
```

Comment:
Appreciate this wasn't described as a requirement in the ticket - but would it be worth considering making this a UUID with a foreign key to an actual Sections model? It feels a bit odd making a comment model without making a model for the object that is being commented? Or is there a ticket for making the sections model?

Reply:
yes, I think this will be the next iteration after our talk on the db structure today. Originally we had the idea that a section would be an INT as per convos with @stevewalton28 on mural [here ](https://app.mural.co/t/mhclg2837/m/mhclg2837/1666017408703/070f1702ce92e818d80fd6af184a258f8ed06768?sender=u7723e74e80c664f54eda3069) but as part of https://digital.dclg.gov.uk/jira/browse/FS-1671 we will be changing this to a separate table with an uuid link as you say 👍